### DISCUSSION TITLE
Invalid Paths When Increasing Depth in Path Computation

### DISCUSSION THREAD

**Discussion by steffenitely** [NVlabs/sionna#626] [Q&A] [answered]
Hi everyone,

I've encountered something unusual when increasing the depth in the computing paths function. I'm reading the number of paths and their delays, but when the depth is increased, the LOS path becomes invalid. This also affects some other paths. I've included pictures to illustrate the issue.

I was wondering if anyone might know what could be causing this. Additionally, there are multiple computed paths that appear to be invalid in general (eg. path 3 in the pictures). I thought invalid paths would only be relevant when simulating with multiple transmitters or receivers, but in my case, I'm only using one of each. 

Here's an example with a depth of 1, where the LOS path (path 0) has a delay of around 10 ns:
![image](https://github.com/user-attachments/assets/1abd6412-e18e-4908-8a0b-93e507ea8973)

When I increased the depth to 2, the LOS path becomes invalid:
![image](https://github.com/user-attachments/assets/77571d8f-f1f9-48cc-949f-b208087a762b)

Thankfull for any help.
Cheers

Edit: I believe the issue is caused by Tx and Rx being perfectly in line with the reflected paths. It makes sense that in these scenarios some paths should be labeld as invalid, but that should not effect the LOS path when the depth is increased.

**Comment by faycalaa** [comment, marked as answer]
Hi @steffenitely,

Thank you for reporting this issue. We are able to reproduce this bug, and it will be fixed in the next release.
